static/app/utils/discover/eventView.tsx

```diff
@@ -413,7 +413,7 @@ class EventView {
       statsPeriod: saved.range,
     });
 
-    return new EventView({
+    const eventView = new EventView({
```

Comment:
nit: is this change needed?

Reply:
oops no this is not needed, left it in by accident during testing. cleaned it up!